omnibus/config/software/gstatus.rb

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Why here and not in omnibus-software repo ?

Reply:
I don't have a preference, the last time I added a dep it ended up being here 😅 . I can move this to omnibus-software repo if preferred!

Reply:
You're right here is fine.